pkg/chains/formats/intotoite6/pipelinerun/pipelinerun.go

```diff
@@ -140,10 +140,6 @@ func buildConfig(pro *objects.PipelineRunObject, logger *zap.SugaredLogger) Buil
 			Results:    tr.Status.TaskRunResults,
 		}
 
-		if t.TaskRef != nil {
-			task.Ref = *t.TaskRef
-		}
-
```

Comment:
Hey @bcaton85 @lcarva ,
This is related to our previous discussion https://github.com/tektoncd/chains/pull/436#discussion_r973143588. Please take a look and let me know what you think. Hope this is not a backward incompatible change.

Reply:
How do I determine which Tasks were used from the PipelineRun attestation? This is a pretty important piece of information that we need.

Reply:
`buildConfig.tasks[i].invocation.configSource` should be able to tell that b/c it includes fields like the uri, digest and entrypoint. For example, [the configsource for in-cluster task](https://github.com/tektoncd/pipeline/pull/5687#issue-1423152223) will be the api path which has the information like namespace, kind, resource name and uid.

Reply:
Is there any disadvantage to leaving this field in?

I'm tempted to leave this as-is:

1. For compatibility (we would probably want a more phased deprecation if we wanted to remove it completely)
2. I feel like there's some additional value in keeping the value the user gave (e.g. user said to checkout `main`, which was resolved to `34f4b3`) even if it doesn't really give us

Reply:
Thank you for sharing your thoughts @wlynch @lcarva. Sorry I didn't know there is a policy looking at this field. @lcarva 

> Is there any disadvantage to leaving this field in?

I don't think there is disadvantage to leave this field in other than sort of duplication with configsource. Keeping this for now SGTM.

Reply:
I'll close this PR for now. We can come back when we need it.